src/vocgui/serializers.py

```diff
@@ -56,15 +56,18 @@ class Meta:
         """
 
         model = AlternativeWord
-        fields = ("alt_word", "article", "plural_article")
+        fields = ("alt_word", "article")
 
     def to_representation(self, data):
         data = super(AlternativeWordSerializer, self).to_representation(data)
-        if data['article'] == "die (Plural)":
-            data['article'] = "die"
-            data['plural_article'] = True
-        else:
-            data['plural_article'] = False
+        if data['article'] == "der":
```

Comment:
I think having the if ... else construct in the code several times is bound to produce mistakes and errors in the long run. IMHO there should be a static map somewhere, that directly maps word to integers.

Reply:
I agree. In fact, if you use the integer representations as keys in the [model field choices](https://github.com/digitalfabrik/lunes-cms/blob/4503e773cd78d7480b0a0ad419bd7409a5b22d3b/src/vocgui/models.py#L30-L36), you can just get the readable representation by `alt_word.get_article_display()` (see [docs](https://docs.djangoproject.com/en/3.2/ref/models/instances/#django.db.models.Model.get_FOO_display)) and don't need to do the conversion manually.

Reply:
Using model field choices is a better way. @svenseeberg Since I changed article to IntegerField, there will be issues with our database.